src/app/pools/components/PoolDeleteForm/PoolDeleteForm.test.tsx

```diff
@@ -1,11 +1,45 @@
 import PoolDeleteForm from "./PoolDeleteForm";
 
-import { renderWithBrowserRouter, screen } from "@/testing/utils";
+import { poolsResolvers } from "@/testing/resolvers/pools";
+import {
+  screen,
+  renderWithBrowserRouter,
+  setupMockServer,
+  userEvent,
+  waitFor,
+} from "@/testing/utils";
 
-it("renders", () => {
-  renderWithBrowserRouter(<PoolDeleteForm id={1} />);
+const mockServer = setupMockServer(poolsResolvers.deletePool.handler());
 
-  expect(
-    screen.getByRole("form", { name: /Confirm pool deletion/i })
-  ).toBeInTheDocument();
+describe("PoolDeleteForm", () => {
+  it("renders", () => {
+    renderWithBrowserRouter(<PoolDeleteForm id={1} />, {
```

Comment:
(optional) you could use `renderWithProviders` here instead of `renderWithBrowserRouter`